boskos/boskos.go

```diff
@@ -155,7 +155,11 @@ func handleAcquire(r *ranch.Ranch) http.HandlerFunc {
 		resource, err := r.Acquire(rtype, state, dest, owner)
 
 		if err != nil {
-			logrus.WithError(err).Errorf("No available resource")
+			if err2, ok := err.(ranch.ResourceNotFound); ok {
+				logrus.WithError(err2).Warningf("No available resource")
```

Comment:
Can we distinguish between "I'll never have anything to do" and "I'm already done?" The latter seems like it should not be a warning

Reply:
I think from server side we cannot identify the difference between `well I'll just do nothing if I don't get a resource` and `Nooo I absolutely need this resource or I'll panic`... We can add an additional param to identify the calls from client side.